### ISSUE TITLE
Write brief description of group goals

### ISSUE THREAD

**Issue by essential-randomness** [Fandom-Coders/fancoders.com#1]

**Comment by cmdonovann** [comment]
Okay so re: group goals, I can write something up and make it sound nice, but not sure if it needs to be a broad overview or if we want to set specific/granular goals for the fancoders group? Example of broad goal (to me) would be something like what we already have in the "Who Are We" section of the index page, like "our goal is to increase coding literacy in fandom and connect fandom coders" etc etc, although we could reword it to be a teeny bit more specific like "our goals are to form communities in which experienced fandom coders can help teach coding skills and code literacy to fandom coding newcomers, to provide opportunities for experienced and inexperienced coders alike to test out their skills in a supportive environment, and to create a network of like-minded fandom coding enthusiasts who can share their knowledge and enthusiasm." HOWEVER if we are looking for more specific/granular goals (something like "teach current members to edit the fancoders.com site," "increase membership in fancoder discord," or something related to a specific project) i think we would wanna discuss that in the discord and outline what our specific goals are. (unless that has already been discussed and i missed it, lol, in that case ignore me)

**Comment by essential-randomness** [comment]
I feel like most of our goals are indeed already hinted at in the "who are we" section, but I'd like to rework the home page so that its copy is more "captivating at a glance". In particular, I feel like the who we are statement right now is hard to skim (though the highlights I added help), and should be more of a way-shorter elevator pitch.

I think for the "our goals" section I'd like to keep it to 3(ish) broad points ("increase coding literacy in fandom", "help connect fandom people with fandom-led open source projects", "strategize about the future of the fandom web"—to brainstorm a quick example), which we can then maybe give practical examples of how we want to lead in another page, to be written together or later.

**Comment by merelydovely** [comment]
brainstorm session came up with:

Fandom Coders aims to:
- increase coding literacy in fandom
- collect and share accessible tutorials and coding resources
- connect fandom builders with each other
- increase the representation of fans in open-source projects
- empower fans to create and shape the fandom internet they want to see!
